docs/src/main/asciidoc/spring-cloud-netflix.adoc

```diff
@@ -69,6 +69,8 @@ eureka:
 
 In the preceding example, "defaultZone" is a magic string fallback value that provides the service URL for any client that does not express a preference (in other words, it is a useful default).
 
+WARNING: defaultZone property doesn't work without camel case because "serviceUrl" property is a Map<String, String>. This means that there is no information available to know that default-url should be defaultUrl.
```

Comment:
How about this?
```
WARNING: The "defaultZone" property is case sensitive and requires camel case because the "serviceUrl" property is a `Map<String, String>`. Therefore, the "defaultZone" property does not follow the normal Spring Boot snake-case convention of "default-zone".
```

Reply:
perfect, thanks